metadata_fetcher/fetchers/ucd_json_fetcher.py

```diff
@@ -89,7 +90,10 @@ def fetch_json_ld(self, url: str) -> Optional[dict]:
         Returns:
             dict
         """
-        response = self.make_http_request(url)
+        session = requests.Session()
+        retries = Retry(total=3, backoff_factor=2)
+        session.mount("https://", HTTPAdapter(max_retries=retries))
+        response = session.get(url=url)
```

Comment:
If you added the backoff_factor to the gloabl configuration for `self.http_session`, then you should be able to do: 

```suggestion
        response = self.http_session.get(url=url)
```

And get rid of lines 93-95 here. 

Sure, `self.http_session` adds some status codes to the forcelist and attached the retry strategy to both http and https, but that seems fine?

Reply:
Yeah, I thought about that too. I figured to introduce the least amount of change possible, but you're right, it probably would be fine.

Reply:
Mmmm, that makes sense - what you did change (`Fetcher.fetch_page()`) is the function that is called like 95% of the time though, so it seemed confusing to me to change it most of the way but not all of the way. Could result in weird errors on strange edge cases where we've had to implement additional requests outside of the standard requests managed by the Fetcher base class.